nbconvert/filters/svg_constants.py

```diff
@@ -0,0 +1,185 @@
+# Via bleach/_vendor/html5lib/filters/sanitizer.py;
```

Comment:
Let's depend on `html5lib` directly instead of inlining.

Reply:
The sanitizer is still deprecated in upstream html5lib.

https://github.com/html5lib/html5lib-python/blob/f7cab6f019ce94a1ec0192b6ff29aaebaf10b50d/html5lib/filters/sanitizer.py#L23-L29

Reply:
For posterity, from the [migration guide](https://github.com/mozilla/bleach/blob/main/docs/migrating.rst): "If you want to stick to the html5lib sanitizer's allow lists, get them from the [sanitizer code](https://github.com/html5lib/html5lib-python/blob/master/html5lib/filters/sanitizer.py). It's probably best to copy them as static lists"

Reply:
Maybe add that comment to the file itself?